de.tudarmstadt.ukp.dkpro.argumentation.nio/src/main/java/de/tudarmstadt/ukp/dkpro/argumentation/annotations/AnnotatedDocument.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) the Department of Informatics, Technische Universität Darmstadt. All Rights Reserved.
+ *
+ * Unauthorized distribution of this file via any medium is strictly prohibited.
+ */
+package de.tudarmstadt.ukp.dkpro.argumentation.annotations;
```

Comment:
@habernal: The classes in `de.tudarmstadt.ukp.dkpro.argumentation.annotations` are mostly used for marshalling/unmarshalling with JSON (cf. e.g. annotations and comments in `de.tudarmstadt.ukp.dkpro.argumentation.annotations.ImmutableSpanTextLabel`): The UIMA types themselves are autogenerated and so I don't want to modify them, and even if I did, they are extremely complicated and so "properly" annotating them for JSON marshalling would be extremely difficult.